### ISSUE TITLE
508-defect-2 [COGNITION]: App Directory - Additional information buttons must have aria-expanded attribute to give screen readers context

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16464]
#  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to understand the Additional Information buttons are expandable, with content hidden inside them.

## Details

**UPDATE:** IE11 + JAWS skips over the expanded content entirely. I have a hunch this is because we're not using the standard Additional Info component, because I've never seen this behavior before. Upgrading this to a severity 2 defect.

Right now assistive tech users do not have context of the buttons showing and hiding content, because there's no `aria-expanded` attribute on the button. It would be good to add that button, remove the HR (or hide it from screen readers) and maybe wrap the shown/hidden content in a DIV. Screenshot and code snippet below. See the [React Additional Info](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/additionalinfo/) component for markup patterns.

## Acceptance Criteria

- [ ] Buttons have an `aria-expanded` attribute that toggles between true and false
- [ ] No axe browser plugin violations
- [ ] Horizontal rule is hidden from assistive tech
- [ ] IE11 + JAWS does not skip the expanded content. Users should be able to press `ENTER` or `SPACE` to expand an accordion, then press `DOWN_ARROW` to read the expanded content block.

## Solution (if known)

```diff
<button
+ aria-expanded="true"
  class="va-button-link ..."
  type="button"
>Learn about Apple Health <i class="fa fa-chevron-up"></i></button>
...
- <hr/>
+ <hr aria-hidden="true" />
```

```diff 
button.va-button-link {
  display: block; /* this helps MacOS VoiceOver not jump focus to the top of the page when opened with SPACE or ENTER */
}
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-19 at 7 54 39 PM](https://user-images.githubusercontent.com/934879/99748880-006b4d00-2aa3-11eb-84a5-127644a83aea.png)

**Comment by 1Copenut** [comment]
LGTM. Moving to close.
